src/frontend/komponenter/Stegflyt/StegKnapp.tsx

```diff
@@ -7,19 +7,19 @@ import { Button, VStack } from '@navikt/ds-react';
 import { useApp } from '../../context/AppContext';
 import { useBehandling } from '../../context/BehandlingContext';
 import { FanePath } from '../../Sider/Behandling/faner';
-import { Steg } from '../../typer/behandling/steg';
+import { Steg, stegErEtterAnnetSteg } from '../../typer/behandling/steg';
 import { RessursStatus } from '../../typer/ressurs';
 import { Feilmelding } from '../Feil/Feilmelding';
 
-export const NesteStegKnapp: FC<{
+export const StegKnapp: FC<{
     nesteFane: FanePath;
     steg: Steg;
     children: React.ReactNode;
```

Comment:
Syntes bruken av denne komponenten ble litt forvirrende nå. Hadde det vært bedre å droppe children her, og heller lage en prop som heter `gåVidereTekst` eller noe sånt? Eventuelt bare latt teksten bli styrt av komponenten selv, ut i fra hvilke steg vi er i. Komponenten overstyrer jo nå uansett det brukeren sender inn med "Rediger steg" i enkelte tilfeller.

Reply:
Teksten vises i det tilfelle man kun skal vise "gå videre". Har strengt tatt ikke endret den funksjonaliteten. 
Er enig om at vi burde kunne bruke en prop som heter `gåVidereTekst` i stedet for å bruke children. Kan gjøre det en egen PR då alle disse stegflyt er litt beroende av hverendre